themes/src/main/resources-community/theme/base/email/messages/messages_nl.properties

```diff
@@ -17,28 +25,43 @@ eventLoginErrorSubject=Inlogfout
+eventRemoveTotpBodyHtml=<p>OTP is verwijderd van uw account om {0} vanuit {1}. Als u dit niet deed, neem dan contact op met uw beheerder.</p>
 eventUpdatePasswordSubject=Wachtwoord gewijzigd
-eventUpdatePasswordBody=Uw wachtwoord is gewijzigd om {0} door {1}. Als u dit niet was, neem dan contact op met uw beheerder.
-eventUpdatePasswordBodyHtml=<p>Uw wachtwoord is gewijzigd om {0} door {1}. Als u dit niet was, neem dan contact op met uw beheerder.</p>
+eventUpdatePasswordBody=Uw wachtwoord is gewijzigd om {0} vanuit {1}. Als u dit niet deed, neem dan contact op met uw beheerder.
+eventUpdatePasswordBodyHtml=<p>Uw wachtwoord is gewijzigd om {0} vanuit {1}. Als u dit niet deed, neem dan contact op met uw beheerder.</p>
 eventUpdateTotpSubject=OTP gewijzigd
-eventUpdateTotpBody=OTP is gewijzigd voor uw account om {0} door {1}. Als u dit niet was, neem dan contact op met uw beheerder.
+eventUpdateTotpBody=OTP is gewijzigd voor uw account om {0} vanuit {1}. Als u dit niet deed, neem dan contact op met uw beheerder.
 eventUpdateTotpBodyHtml=<p>OTP is gewijzigd voor uw account om {0} door {1}. Als u dit niet was, neem dan contact op met uw beheerder.</p>
+eventUpdateCredentialSubject=Aanmeldgegevens bijwerken
+eventUpdateCredentialBody=Uw aanmeldgegevens bij {0} zijn gewijzigd op {1} vanuit {2}. Als u dit niet was, neem dan contact op met uw beheerder.
+eventUpdateCredentialBodyHtml=<p>Uw aanmeldgegevens bij {0} zijn gewijzigd op {1} vanuit {2}. Als u dit niet was, neem dan contact op met uw beheerder.</p>
+eventRemoveCredentialSubject=credential verwijderd
+eventRemoveCredentialBody=Credential {0} is verwijderd van uw account op {1} vanuit {2}. Als u dit niet was, neem dan contact op met uw beheerder.
+eventRemoveCredentialBodyHtml=<p>Credential {0} is verwijderd van uw account op {1} vanuit {2}. Als u dit niet was, neem dan contact op met uw beheerder.</p>
+eventUserDisabledByTemporaryLockoutSubject=Gebruiker tijdelijk geblokkeerd
+eventUserDisabledByTemporaryLockoutBody=Uw gebruiker is tijdelijk geblokkeerd door meerdere meerder gefaalde inlogpogingen op {0}. Indien nodig, kan u een beheerder contacteren.
+eventUserDisabledByTemporaryLockoutHtml=<p>Uw gebruiker is tijdelijk geblokkeerd door meerdere meerder gefaalde inlogpogingen op {0}. Indien nodig, kan u een beheerder contacteren.</p>
+eventUserDisabledByPermanentLockoutSubject=Gebruiker permanent geblokkeerd
+eventUserDisabledByPermanentLockoutBody=Uw gebruiker is permanent geblokkeerd door meerdere meerder gefaalde inlogpogingen op {0}. Gelieve een beheerder te contacteren.
+eventUserDisabledByPermanentLockoutHtml=<p>Uw gebruiker is permanent geblokkeerd door meerdere meerder gefaalde inlogpogingen op {0}. Gelieve een beheerder te contacteren.</p>
 
+requiredAction.CONFIGURE_TOTP=Configureer OTP
+requiredAction.TERMS_AND_CONDITIONS=Accepteer algemene voorwaarden
+requiredAction.UPDATE_PASSWORD=Update wachtwoord
```

Comment:
Although a popular anglicism, perhaps 'bijwerken' would be more appropriate here than 'update'.

Reply:
Sounds good.